DependencyInjection/HttplugExtension.php

```diff
@@ -316,15 +316,15 @@ private function configureClient(ContainerBuilder $container, $clientName, array
             $container
                 ->register($serviceId.'.client', HttpClient::class)
                 ->setFactory([new Reference($arguments['factory']), 'createClient'])
-                ->addArgument($arguments['config'])
-                ->setPublic(false);
```

Comment:
Shouldn't this definition still be private?

Reply:
Sharp, undid this change.